mime-types-data.gemspec

```diff
@@ -19,6 +19,13 @@ Gem::Specification.new do |s|
   s.required_ruby_version = Gem::Requirement.new(">= 2.0".freeze)
   s.rubygems_version = "3.0.3".freeze
   s.summary = "mime-types-data provides a registry for information about MIME media type definitions".freeze
+  s.metadata = {
```

Comment:
The `gemspec` file is an output file, not an input file. See https://github.com/mime-types/ruby-mime-types/pull/138/files and https://github.com/seattlerb/hoe/pull/86. 

The full list of links supported by Hoe is in [lib/hoe.rb:114–123](https://github.com/seattlerb/hoe/blob/master/lib/hoe.rb#L114-L123).

These changes should be in the README.